src/include/pixels/RGB-pixel.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "pixel.hpp"
+
+class RGBPixel : public Pixel
+{
+public:
+	RGBPixel(unsigned char r, unsigned char g, unsigned char b);
+
+public:
+	void apply(const Operation& operation) override;
+
+public:
+	unsigned char r() const;
+	void r(unsigned char);
```

Comment:
Why not set. I'm not sure how much I like to have setters and getters with the same names. Add a name of the parameter.